src/TwigComponent/src/StaticComponent.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Symfony\UX\TwigComponent;
+
+class StaticComponent
```

Comment:
```suggestion
class AnonymousComponent
```

? I see you are using the term _static_ now - are you wanting to use this term instead of _anonymous_?

And we should mark this as internal.